.github/workflows/build-images.yml

```diff
@@ -0,0 +1,91 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+  release:
+  workflow_dispatch:
+
+jobs:
+  build-docker-image:
+    name: Image Build ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: Debian
+            file: Docker/Dockerfile
+            flavor: |
+              latest=auto
+              suffix=
+              onlatest=false
+          - name: Alpine
+            file: Docker/Dockerfile-Alpine
+            flavor: |
+              latest=auto
+              suffix=-alpine
+              onlatest=true
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+      # - name: Login to DockerHub
+      #   uses: docker/login-action@v3
+      #   with:
+      #     username: ${{ secrets.DOCKERHUB_USERNAME }}
+      #     password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # - name: Login to GitHub Container Registry
```

Comment:
Any comments about how pros and cons of (also) using the GitHub Container Registry?
Probably seems overkill for now?

Reply:
For podman user?
And Dockerhub had in the pass some weird decisions I think.

Personaly I prefer the github registry

Reply:
Ok. Let's keep it but comment out for now, so we can easily activate if needed.
I am not familiar with the use-case: Can't Podman users pull from Docker Hub?

Reply:
It can, but the default registry (dockerhub) is not set.

Reply:
Newer podman versions try dockerhub on unspecified image tags. Personally I don't think it matters much as its the same image